test/scripts/e2e-kind.sh

```diff
@@ -45,7 +48,42 @@ should set default value on new IngressClass
 should prevent Ingress creation if more than 1 IngressClass marked as default
 "
 
-SKIPPED_TESTS=$(echo "${SKIPPED_TESTS}" | sed -e '/^\($\|#\)/d' -e 's/ /\\s/g' | tr '\n' '|' | sed -e 's/|$//')
+IPV4_ONLY_TESTS="
+# Limit the IPv4 related test to IPv4 only deployments
+#  See: https://github.com/leblancd/kube-v6-test
```

Comment:
this is outadated, I've removed that limitation in k8s 1.16

Reply:
Per conversation on slack. `[Feature:Networking-IPv4]' tag is for IPv4 tests and to be skipped for IPv6, and `[Feature:Networking-IPv6]' tag is for IPv6 tests and to be skipped for IPv4. Turns out there is currently on one test with each of the tags.